pkg/controller/provider/container/ova/model.go

```diff
@@ -228,3 +229,35 @@ func (r *DiskAdapter) GetUpdates(ctx *Context) (updates []Updater, err error) {
 	}
 	return
 }
+
+type StorageAdapter struct {
+	BaseAdapter
+}
+
+func (r *StorageAdapter) GetUpdates(ctx *Context) (updates []Updater, err error) {
+	return
+}
+
+// List the collection.
+func (r *StorageAdapter) List(ctx *Context) (itr fb.Iterator, err error) {
+	storageList := []Storage{}
+	storageList = append(storageList, Storage{
+		Name: "VM storage",
```

Comment:
Any others suggestion for the name + ID.
They will be the same for all OVA providers and we will have one created for each.

Reply:
why do we need the storage to be the same for all OVA providers?
can't we go with something like: "Storage of provider <name of the provider>" and use the UID of the provider for the storage as well?

Reply:
just to simplify it for users, instead of searching the correct value for each provider use something more unified

Reply:
users are not supposed to write yaml files with our new beautiful UI :) scripts interact with the system through such files and they're expected to query the available storage types before setting something so I don't see this as a goal to make it easy to manually write a mapping configuration
but we can as well go with something like "All disks in OVAs" or something like that